Makefile

```diff
@@ -115,9 +115,9 @@ CUSTOMENVTEST = $(LOCALBIN)/k8s/1.28.7-linux-amd64
 $(CUSTOMENVTEST):
 	if [ ! -d "$(CUSTOMENVTEST)" ]; then \
 		mkdir -p $(LOCALBIN)/k8s/1.28.7-linux-amd64; \
-		curl -L https://sertaccdn.azureedge.net/kube-vap-fix/etcd --output $(LOCALBIN)/k8s/1.28.7-linux-amd64/etcd && chmod +x $(LOCALBIN)/k8s/1.28.7-linux-amd64/etcd; \
-		curl -L https://sertaccdn.azureedge.net/kube-vap-fix/kube-apiserver --output $(LOCALBIN)/k8s/1.28.7-linux-amd64/kube-apiserver && chmod +x $(LOCALBIN)/k8s/1.28.7-linux-amd64/kube-apiserver; \
-		curl -L https://sertaccdn.azureedge.net/kube-vap-fix/kubectl --output $(LOCALBIN)/k8s/1.28.7-linux-amd64/kubectl && chmod +x $(LOCALBIN)/k8s/1.28.7-linux-amd64/kubectl; \
+		curl -L https://sertaccdnvs.azureedge.net/kube-vap-fix/etcd --output $(LOCALBIN)/k8s/1.28.7-linux-amd64/etcd && chmod +x $(LOCALBIN)/k8s/1.28.7-linux-amd64/etcd; \
```

Comment:
As discussed on slack, lets see if we can use upstream kubebuilder instead of host our own.

Reply:
let's merge this to unblock CI asap, then i'll open a PR to revert these changes